docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1361,6 +1361,10 @@ How to **fix** it:
 
 - See [`MimirIngesterReachingSeriesLimit`](#MimirIngesterReachingSeriesLimit) runbook.
 
+### err-mimir-ingester-max-ephemeral-series
+
+This error refers to experimental ephemeral storage in ingesters. Documentation TBD.
```

Comment:
Instead of "Documentation TBD", what if you copy-paste the content of the persistent counterpart runbook and just replace the config names?

Reply:
OK.

Reply:
Done here: https://github.com/grafana/mimir/pull/4004/commits/083b57f43f5368dd7a142981e9627d2999597498